pkg/agent/config/config.go

```diff
@@ -74,6 +76,19 @@ func GetConfig(mode Mode) (*Config, error) {
 	return config, nil
 }
 
+func setLogLevel() {
+	agentLogLevel := os.Getenv("AGENT_LOG_LEVEL")
+	if agentLogLevel != "" {
+		level, err := log.ParseLevel(agentLogLevel)
+		if err != nil {
+			log.Errorf("Invalid log level: %s, defaulting to info level", agentLogLevel)
```

Comment:
You're missing actually defaulting to info here 😅 

`level = log.InfoLevel`

Reply:
Sorry sorry... Added it!

Reply:
do it as I said `level = log.InfoLevel`

Your way you're setting the log level twice which is unnecessary.

Reply:
Get it!